### DISCUSSION TITLE
Send map coordinates to car

### DISCUSSION THREAD

**Discussion by Nico04** [flobz/psa_car_controller#928] [Ideas]
With the official app you can send/share a map address/point to the car, so you can with the built-in navigation system. It's quite handy, but it doesn't work very well. Is that possible to implement in this project?

**Comment by GeraldPape** [comment]
I don't think so. At least for my C5 Aircross this works using Bluetooth from the mobile device/app.

**Reply by Nico04** [reply]
In my e208 it's possible, as I have done it several times. But not sure from official API though.
But there is a feature about that in the doc : https://github.com/Stellantis/stellantis.github.io/blob/b4c046be980209cf0bf48cb0911ae6765a5f5be8/assets/openapi/api-b2c.yaml#L1049
